src/interface/others/ItemDamageTaken.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class ItemDamageTaken extends React.PureComponent {
+  static propTypes = {
+    amount: PropTypes.number.isRequired,
+    approximate: PropTypes.bool,
+  };
+  static contextTypes = {
+    parser: PropTypes.object.isRequired,
+  };
+
+  render() {
+    const { amount, approximate } = this.props;
+    const { parser } = this.context;
+
+    return (
+      <>
+        <img
+          src="/img/damage-taken.png"
```

Comment:
Please use the shield icon for damage taken as I reckon it has been used in that context previously.